utils/spec/tsconfig.json

```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "../../tsconfig.spec.json"
```

Comment:
Any reason you removed this guy?

Reply:
I removed the folder since there were no specs left

Reply:
Hah - thanks 😅